src/controllers/UserController.js

```diff
@@ -184,6 +184,43 @@ class UserController {
       return HelperMethods.serverError(res);
     }
   }
+
+  /**
+  * Verify a user's email
+  * Route: POST: /auth/verify_email
+  * @param {object} req - HTTP Request object
+  * @param {object} res - HTTP Response object
+  * @return {res} res - HTTP Response object
+  * @memberof UserController
+ */
+  static async updatedUser(req, res) {
+    const token = req.headers['x-access-token'];
```

Comment:
There is a middleware function called `checkToken` that checks and verifies the token before passing the request to the controller. this function adds the payload to `req.decoded` so on line 197, you would write `const payload = req.deocded;`